modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -34,7 +34,9 @@ When you create a Dedicated cluster, you select the supported xref:deploy:deploy
 
 ==== Sign up for Dedicated
 
-To start using Dedicated, contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda sales^] to request a private offer for monthly or annual committed use. With a usage-based billing commitment, you sign up for a monthly or an annual minimum spend amount. You can then use xref:deploy:deployment-option/cloud/manage-billing/gcp-commit.adoc[Google Cloud Marketplace] or xref:deploy:deployment-option/cloud/manage-billing/aws-commit.adoc[AWS Marketplace] to quickly provision Dedicated Cloud clusters, and you can view invoices and manage your subscription in the marketplace.
+To start using Dedicated, sign up for a free trial on the AWS Marketplace. New trials receive $300 (USD) in free credits to spend in the first 30 days. This should be enough to run Redpanda with reasonable throughput. No credit card is required for a trial. To continue using Dedicated, add a credit card and pay as you go.
```

Comment:
AFAIK credit card isn't applicable for marketplace trials. Marketplace sign-ups pay via their cloud account (we charge them via marketplace). Not sure whether we mandate the payment method they use in the marketplace.

Also, no action is needed to extend the trial. Once they burn free credits or the free credits expire, they are just charged via marketplace. They are charged unless they cancel the subscription. 

@towfiqa to answer.

Reply:
Thanks, @vladoschreiner! I've updated this and added to the [What's New for Cloud](https://deploy-preview-612--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/whats-new-cloud/).